src/patches/vs1053b-patches-flac.plg

```diff
@@ -0,0 +1,1124 @@
+/* User application code loading tables for VS10xx */
+#if 0
+void LoadUserCode(void) {
```

Comment:
As earlier mentioned, do we really need to include `LoadUserCode(void)` everywhere which we will not use?
Same for all other `*.plg` files.